server/app/views/applicant/ApplicantLayout.java

```diff
@@ -394,10 +400,12 @@ private DivTag authDisplaySection(
           div(
               span(loggedInAsMessage).withClasses("text-sm"),
```

Comment:
**suggestion:** I think it should not show the loggedInAsMessage if you don't have a session. I remember when I first used CiviForm I thought it was weird that clicking "End session" had no visible effect, and I think it would make more sense if it didn't show "You're logged in" until you actually start a session.

Same suggestion for the Northstar path; I didn't dig up the code for it.